kubernetes/thelounge.yml.erb

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: service
+spec:
+  selector:
+    app: thelounge
+  ports:
+    - port: 80
+      targetPort: 9000
+--
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: deployment
+  labels:
+    app: thelounge
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: thelounge
+  template:
+    metadata:
+      labels:
+        app: thelounge
+    spec:
+      containers:
+        - name: thelounge
+          image: "tonylianlong/thelounge:1.0.0"
```

Comment:
I believe you can use `image: "docker.ocf.berkeley.edu/thelounge:<%= version %>"` here instead, since the [first build](https://jenkins.ocf.berkeley.edu/job/thelounge/job/master/1/) has passed and uploaded the image.